contracts/MetaDataService.sol

```diff
@@ -0,0 +1,15 @@
+pragma solidity >=0.8.4;
+import "@openzeppelin/contracts/utils/Strings.sol";
+
+contract MetaDataService {
```

Comment:
Nit: should be `MetadataService`.

Reply:
Also, this should probably have a descriptive name like `StaticMetadataService`.